source/frontend/src/components/maps/leaflet/Control/Search/SearchView.tsx

```diff
@@ -0,0 +1,132 @@
+import { Feature, Geometry } from 'geojson';
+import { chain } from 'lodash';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+import { MapFeatureData } from '@/components/common/mapFSM/models';
+import { Scrollable } from '@/components/common/Scrollable/Scrollable';
+import { Section } from '@/components/common/Section/Section';
+import { PropertyFilter } from '@/features/properties/filter';
+import {
+  defaultPropertyFilter,
+  IPropertyFilter,
+} from '@/features/properties/filter/IPropertyFilter';
+import { PMBC_FullyAttributed_Feature_Properties } from '@/models/layers/parcelMapBC';
+import { PIMS_Property_Location_View } from '@/models/layers/pimsPropertyLocationView';
+import { exists } from '@/utils';
+import { isStrataCommonProperty } from '@/utils/propertyUtils';
+
+export interface ISearchViewProps {
+  onFilterChange: (filter: IPropertyFilter) => void;
+  propertyFilter: IPropertyFilter;
+  searchResult: MapFeatureData;
+}
+
+interface PropertyProjection<T> {
+  isStrataLot: boolean;
+  pid: string | null;
+  pin: string | null;
+  plan: string | null;
+  feature: Feature<Geometry, T> | null;
+}
+
+export const SearchView: React.FC<ISearchViewProps> = props => {
+  const groupedFeatures = chain(props.searchResult?.fullyAttributedFeatures.features)
+    .groupBy(feature => feature?.properties?.PLAN_NUMBER)
+    .map(
+      planGroup =>
+        planGroup
+          ?.map<PropertyProjection<PMBC_FullyAttributed_Feature_Properties>>(x => ({
+            pid: x.properties.PID_FORMATTED,
+            pin: exists(x.properties.PIN) ? String(x.properties.PIN) : null,
+            isStrataLot: isStrataCommonProperty(x),
+            feature: x,
+            plan: x.properties.PLAN_NUMBER,
+          }))
+          .sort((a, b) => {
+            if (a.isStrataLot === b.isStrataLot) return 0;
+            if (a.isStrataLot) return -1;
+            if (b.isStrataLot) return 1;
+            return 0;
+          }) ?? [],
+    );
+
+  const propertyProjections = groupedFeatures.value().flatMap(x => x) ?? [];
+
+  const pimsGroupedFeatures = chain(props.searchResult?.pimsLocationFeatures.features)
+    .groupBy(feature => feature?.properties?.SURVEY_PLAN_NUMBER)
+    .map(
+      planGroup =>
+        planGroup
+          ?.map<PropertyProjection<PIMS_Property_Location_View>>(x => ({
+            pid: x.properties.PID_PADDED,
+            pin: exists(x.properties.PIN) ? String(x.properties.PIN) : null,
+            isStrataLot: false,
```

Comment:
Am I missing something here? it seems like you are setting isStrataLot to false and then sort by is isStrataLot, isn't that a no-op?